.changeset/rotten-walls-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/abi-coder": minor
```

Comment:
```suggestion
```

Test changes don't require a package redeployment as it is not shipped with the build.